### PR TITLE
Add query parameter to restrict origin

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

**Comment by ara4n** [comment]
lgtm. whilst you're in there, perhaps put a comment to explain wtf this file is and what's going on, and link to the synapse instructions of how to host your own, and perhaps the github issue for getting rid of it?

File: README.md [hunk 1]
```diff
@@ -4,16 +4,10 @@ UserContent
 A way to render user generated content with a different origin to the main application.
 This can be used to avoid XSS attacks.
 
-Version 1
----------
+This is used by Riot to display content in end-to-end encrypted chats. See
```
Comment (ara4n) [inline README.md]: "to display the download button for encrypted attachments" perhaps?

File: v1.html [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
 <head>
+<!--
+Hello! If you're reading this, perhaps you're wondering what this
+file is doing and why your Riot is using it.
+
+In short, this allows Riot to isolate potentially unsafe encrypted
+attachments into their own origin, away from your Riot. See
+https://github.com/matrix-org/usercontent/blob/master/README.md
+for more info, or https://github.com/vector-im/riot-web/blob/master/README.md#configjson
+if you'd like to host your own.
```
Comment (ara4n) [inline v1.html]: let's add a link to bug 6173 so people can track progress on replacing it with iframes

**Review by ara4n — CHANGES_REQUESTED** [review]

**Comment by dbkr** [comment]
@ara4n PTAL

**Pull request merged**